plugins/org.jkiss.dbeaver.ext.generic/src/org/jkiss/dbeaver/ext/generic/edit/GenericPrimaryKeyManager.java

```diff
@@ -71,8 +84,39 @@ protected GenericUniqueKey createDatabaseObject(
             false);
     }
 
+    @Override
+    protected void addObjectCreateActions(
+        DBRProgressMonitor monitor,
+        DBCExecutionContext executionContext,
+        List<DBEPersistAction> actions,
+        ObjectCreateCommand command,
+        Map<String, Object> options
+    ) {
+        GenericUniqueKey key = command.getObject();
+        if (key.getConstraintType() == DBSEntityConstraintType.CHECK && (key instanceof DBSTableCheckConstraint check)) {
```

Comment:
Redundant parenthesis
```suggestion
        if (key.getConstraintType() == DBSEntityConstraintType.CHECK && key instanceof DBSTableCheckConstraint check) {
```